register_users.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+#!/usr/bin/python
+
+import urllib2, random, requests, urllib2, socket, socks, sys
+from bs4 import BeautifulSoup
+from time import sleep
```

Comment:
Unused import. Please use pylint or some other python linter to find out such obvious issues before committing.